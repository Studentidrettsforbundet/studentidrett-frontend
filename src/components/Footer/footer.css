.footer{
    background-color: #26366F;
    width: 100%;
    padding: 1rem 0;
}

.footerContainer {
    display: flex;
    flex-direction: row;
    font-weight: 300;
    color: white;
    justify-content: space-between;
}

a {
    color: white !important;
}

.footerLead {
    font-weight: 500;
    font-size: 1.175rem;
}

@media only screen and (max-width: 720px) {
    .footerContainer {
        flex-direction: column;
        text-align: center;
    }
    .footerSection{
        margin: 8px 0;
    }
}