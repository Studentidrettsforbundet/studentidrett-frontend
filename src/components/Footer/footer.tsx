import React from 'react';
import './footer.css';

const Footer = (): JSX.Element => (
    <div className="footer">
        <div className="container footerContainer">
            <section className="footerSection">
                <span className="footerLead">Telefon</span>
                <br />
                <a href="tel:+47 976 18 966" data-content="+47 976 18 966" data-type="phone">
                    976 18 966
                </a>
            </section>
            <section className="footerSection">
                <span className="footerLead">E-mail</span>
                <br />
                <a href="mailto:nsi@studentidrett.no">nsi@studentidrett.no</a>
            </section>
            <section className="footerSection">
                <span className="footerLead">Addresse</span>
                <br />
                Idrettens Hus Sognsveien 73
                <br />
                0840 Oslo
            </section>
        </div>
    </div>
);

export default Footer;
