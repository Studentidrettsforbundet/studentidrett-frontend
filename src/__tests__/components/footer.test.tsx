import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../../components/footer';

describe('resultPage', () => {
    describe('snapshot tests', () => {
        it('should render correctly', () => {
            const tree = renderer.create(<Footer />).toJSON();
            expect(tree).toMatchSnapshot();
        });
    });
});
