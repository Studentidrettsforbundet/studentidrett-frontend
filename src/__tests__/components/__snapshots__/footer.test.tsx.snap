// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`resultPage snapshot tests should render correctly 1`] = `
<div
  className="footer"
/>
`;
